packages/openapi-sampler/src/samplers/StringSampler.ts

```diff
@@ -55,30 +55,60 @@ export class StringSampler implements Sampler {
     const randExp = new RandExp(pattern);
 
     if (min) {
```

Comment:
Use more explicit undefined checks for min and max to avoid potential issues with 0. Here is example:
```suggestion
    if (min !== undefined) {
```